packages/app-page-builder/src/editor/plugins/elements/menu/Menu.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from "react";
+import { ElementRoot } from "@webiny/app-page-builder/render/components/ElementRoot";
+import { get } from "lodash";
+import styled from "@emotion/styled";
+import { connect } from "react-redux";
+import { getPlugins } from "@webiny/plugins";
+import { getActiveElementId } from "@webiny/app-page-builder/editor/selectors";
+import { PbElement } from "@webiny/app-page-builder/types";
+
+import { PbPageElementMenuComponentPlugin } from "@webiny/app-page-builder/types";
+
+const Overlay = styled("div")({
+    background: "black",
+    position: "absolute",
+    width: "100%",
+    height: "100%",
+    zIndex: 10,
+    opacity: 0.25
+});
+
+export type MenuElementProps = {
+    isActive: boolean;
+    element: PbElement;
+};
+
+const Menu = (props: MenuElementProps) => {
+    //menu data passed through here
+    const { element, isActive } = props;
+    console.log("MENU PROPS ::::;;;;");
+    console.log(props);
+ 
+    const data = get(element, "data") || {};
```

Comment:
Use optional instead of get if you are unsure that element actually is a object. So it looks like this: `const data = element?.data || {}`;
From MenuElementProps it is notable that element must be a PbElement and it has data property. So, basically, it would be enough to write `const data = element.data;`.

Reply:
@brunozoric Im sure its an object, so I am just going to call get and remove ```|| {} ;```